CONTRIBUTORS.md

```diff
@@ -0,0 +1,33 @@
+# Contributors
+
+- Alexander Knipping ([@obitech](https://github.com/obitech))
+- Amauri Bizerra ([@AmauriAires](https://github.com/AmauriAires))
+- anirudh-modi ([@anirudh-modi](https://github.com/anirudh-modi))
+- ENT8R ([@ENT8R](https://github.com/ENT8R))
+- Flo Becker ([@beckerei](https://github.com/beckerei))
+- Josh Cooper ([@JoshCooperr](https://github.com/JoshCooperr))
+- Linus Brogan ([@linusaurusrex](https://github.com/linusaurusrex))
+- Metta Ong ([@ongspxm](https://github.com/ongspxm))
+- Michael Corwin ([@mcorwin22](https://github.com/mcorwin22))
+- OogieBoogieInJSON ([@OogieBoogieInJSON](https://github.com/OogieBoogieInJSON))
+- redagavin ([@redagavin](https://github.com/redagavin))
+- xlomonosvx ([@xlomonosvx](https://github.com/xlomonosvx))
```

Comment:
If there is only the username, you do not need to mention it twice. In this case, you can really directly link it via `[@user](…)`. (taht looks strange IMHO)